mappings/markdown/Zeek/conn_log/README.md

```diff
@@ -1,55 +1,73 @@
-|'packets_in'                   |'orig_pkts'    |
-|'orig_bytes.ip'                |'orig_ip_bytes'|
-|'bytes_out'                    |'resp_bytes'   |
-|'packets_out'                  |'resp_pkts'    |
-|'resp_bytes.ip'                |'resp_ip_bytes'|
-|'duration'                     |'duration'     |
-|'unmapped'                     |'conn_state'   |
-
- ### Conditional Mapping:
- - Any fields described within the conditional mappings are subject to dynamic mappings contingent on a conditional evaluation of source data. Fields which fail to meet a particular conditional are assigned a default value from the OCSF schema description.
-| OCSF                 | Raw             | Notes                       |
-| ---------------------| ----------------| ----------------------------|
+|`time`                         |`ts`           |
+|`uuid`                         |`uid`          |
+|`src_endpoint.ip`              |`id.orig_h`    |
+|`src_endpoint.port`            |`id.orig_p`    |
+|`src_endpoint.location.country`|`orig_cc`      |
+|`src_endpoint.mac`             |`orig_l2_addr` |
+|`dst_endpoint.ip`              |`id.resp_h`    |
+|`dst_endpoint.port`            |`id.resp_p`    |
+|`dst_endpoint.location.country`|`resp_cc`      |
+|`dst_endpoint.mac`             |`resp_l2_addr` |
+|`app_name`                     |`service`      |
+|`connection_info.protocol_name`|`proto`        |
+|`duration`                     |`duration`     |
+|`network_endpoint.vlan_uid`    |`vlan`         |
+|`traffic.bytes_in`             |`resp_bytes`   |
+|`traffic.packets_in`           |`resp_pkts`    |
+|`traffic.bytes_out`            |`orig_bytes`   |
+|`traffic.packets_out`          |`orig_pkts`    |
+|`traffic.bytes`                |`resp_bytes` + `orig_bytes` |
+|`traffic.packets`              |`orig_pkts` + `resp_pkts`   |
 
+ ### Unmapped (proposed):
 
- ### Proposed New OCSF attributes:
- - 
-| OCSF                 | Raw             | Notes                       |
-| ---------------------| ----------------| ----------------------------|
-|'application_protocol'|'service'        |'propose new ocsf field name'|
-|'bytes_missed'        |'missed_bytes'   |'propose new ocsf field name'|
-|'connection_history'  |'history'        |'propose new ocsf field name'|
+| OCSF                 | Raw             |
+| ---------------------| ----------------| 
+|`src_endpoint.observables.type_id:1 && src_endpoint.observables.reputation = id.orig_h_name.src` |`id.orig_h_name.vals` |
+|`dst_endpoint.observables.type_id:1 && dst_endpoint.observables.reputation = id.orig_h_name.src` |`id.orig_h_name.vals` |
+|`(bytes_missed)`        |`missed_bytes` |
+|`(connection_info.history)`  |`history`      |
+|`(connection_info.state)`    |`conn_state`   |
+|`unmapped`                   |`app`          |
+|`unmapped`                   |`community_id` |
+|`unmapped`                   |`corelight_shunted`   |
+|`unmapped`                   |`pcr`          |
```

Comment:
Some of these—like `pcr`—are fancy extensions via 3rd-party packages. Since Zeek logs can be infinitely extended, we could consider mentioning what package adds what logs. Or we simply restrict ourselves to the OOTB fields.